pdns/dnsdistdist/docs/advanced/ebpf.rst

```diff
@@ -73,3 +73,6 @@ Since 1.6.0, the default BPF filter set via :func:`setDefaultBPFFilter` will aut
 That feature might require an increase of the memory limit associated to a socket, via the sysctl setting ``net.core.optmem_max``.
 When attaching an eBPF program to a socket, the size of the program is checked against this limit, and the default value might not be enough.
 Large map sizes might also require an increase of ``RLIMIT_MEMLOCK``, which can be done by adding ``LimitMEMLOCK=infinity`` in the systemd unit file.
+
+To change the default hard limit on ``RLIMIT_MEMLOCK`` add the following line to ``/etc/security/limits.conf`` for the user
+  > $USER   hard    memlock   1073741824
```

Comment:
You might have seen the spell-checker warning about this line, would you mind adding "memlock" to the list of allowed words in `.github/actions/spell-check/expect.txt`? It looks like it's already there but in caps, so I think we need to replace the existing "MEMLOCK" line by "memlock".

Reply:
Adding it